src/iop/dither.c

```diff
@@ -304,6 +305,10 @@ static int _get_dither_parameters(
       graymode = 0;
       *levels = 16;
       break;
+    case DITHER_FS6BIT_GRAY:
+      graymode = 1;
+      *levels = MAX(64, MIN(63 * bds + 1, 256));
```

Comment:
This forces at least 64 levels, but allows up to 256 on downscaled display....  At the lower bit depths, this approach gives a better visual approximation of the result, but I'm not sure there's any need for that when you already have 64 gray levels.

Reply:
Not sure either but it wouldn't hurt. (Unfortunately i don't have access to one of the pure-gray monitors of our radiologists :-)